src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -2401,7 +2325,8 @@ static void VehiclePitchUp150(
     if (vehicleEntry->sprite_flags & VEHICLE_SPRITE_FLAG_VERTICAL_SLOPES)
     {
         int32_t boundingBoxNum = (YawTo4(imageDirection)) + 88;
-        int32_t spriteNum = (((YawTo4(imageDirection)) + 96) * vehicleEntry->base_num_frames)
+        int32_t spriteNum = (((YawTo4(imageDirection)) + 32 + vehicleEntry->NumRotationFrames() * 2)
```

Comment:
are you sure it shouldn't be NumRotationFrames()*3 and drop the +32 or am i misunderstanding how this works.

Reply:
The sprites that are not vertical are fixed at 4 frames per rotation